receiver/splunkenterprisereceiver/client.go

```diff
@@ -0,0 +1,114 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package splunkenterprisereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver"
+
+import (
+	"crypto/tls"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type splunkEntClient struct {
+	endpoint  *url.URL
+	client    *http.Client
+	basicAuth string
+}
+
+func newSplunkEntClient(cfg *Config) splunkEntClient {
+	// tls party
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
can we get this from the config instead? Does it have to be insecure always? Not a blocker but needs a follow up issue